welcome/src/main/kotlin/edu/artic/welcome/WelcomeFragment.kt

```diff
@@ -19,6 +27,13 @@ class WelcomeFragment : BaseViewModelFragment<WelcomeViewModel>() {
     override val layoutResId: Int
         get() = R.layout.fragment_welcome
 
+    @Inject
+    lateinit var moshi: Moshi
+
+    var welcomePreferencesManager: WelcomePreferencesManager? = null
```

Comment:
This should be injected. We can inject an instance of `Context` in an `AppModule` which provides the application context to our preferences manager. Then injecting it will suffice.

Reply:
Done.